### PR TITLE
Tabs Component - Add initialise after mount

### PR DESCRIPTION
**Pull request by alextrastero**
related to https://github.com/react-materialize/react-materialize/issues/70

Tabs now are initialised after mount

### CODE REVIEW

File: src/Tabs.js [hunk 1]
```diff
@@ -1,43 +1,74 @@
 import React from 'react';
 import cx from 'classnames';
+import idgen from './idgen';
 
 import Row from './Row';
 import Col from './Col';
 
 class Tabs extends React.Component {
+  componentDidMount() {
+    if (typeof $ !== 'undefined') {
```
Comment (wedneyyuri) [inline src/Tabs.js]: If jQuery is not defined, then it will fail silently.

Comment (alextrastero) [inline src/Tabs.js]: Yup, i'm afraid this happens in many places for now...

Comment (alextrastero) [inline src/Tabs.js]: Would need to Throw an exception

Comment (wedneyyuri) [inline src/Tabs.js]: You don't need to explicitly throw an exception:

image

Just believe that jQuery will be available on the page :)

**Comment by alextrastero** [comment]
WIP, still checking some stuff, but as @wedneyyuri mentioned, it's best to show an error if there is an error.

**Comment by isaiah** [comment]
@alextrastero looks good to me, what's left here?

**Comment by alextrastero** [comment]
I need to have another look in the w/e

Alex

**Comment by reesmanp** [comment]
PR #197 might be relevant here.

**Comment by alextrastero** [comment]
Please try `react-materialize3.x` and reopen if this is still happening, thanks!

**Pull request closed without merging**